resources/assets/components/utilities/NewsletterSubscription/NewsletterSubscriptionForm.js

```diff
@@ -0,0 +1,120 @@
+import fetch from 'node-fetch';
+import React, { useState } from 'react';
+
+import { env } from '../../../helpers/env';
+import { NEWSLETTER_TOPICS } from './config';
+import PrimaryButton from '../Button/PrimaryButton';
+import { isAuthenticated } from '../../../helpers/auth';
+import NewsletterSubscriptionCard from './NewsletterSubscriptionCard';
+import NewsLetterSubscriptionFormInput from './NewsletterSubscriptionFormInput';
+
+const NewsletterSubscriptionForm = () => {
+  const [subscriptions, setSubscription] = useState([]);
```

Comment:
nit, but should the update method be pluralized as well?

```suggestion
  const [subscriptions, setSubscriptions] = useState([]);
```

Reply:
yes! good 👁️